WalletWasabi.Gui/MainWindow.xaml.cs

```diff
@@ -44,32 +50,35 @@ public MainWindow()
 		}
 
 		private Global Global { get; }
+		public TerminateService TerminateService { get; }
 
 		private void InitializeComponent()
 		{
 			AvaloniaXamlLoader.Load(this);
 		}
 
 		private int _closingState;
```

Comment:
```suggestion
                /// <summary>All access requires use of <see cref="Interlocked"/> class.</summary>
		private int _closingState;
```

Reply:
This might be not true https://github.com/zkSNACKs/WalletWasabi/pull/4670#discussion_r518039819